modules/ztp-preparing-the-hub-cluster-for-ztp.adoc

```diff
@@ -1,26 +1,24 @@
 // Module included in the following assemblies:
 //
-// *scalability_and_performance/ztp-deploying-disconnected.adoc
+// *scalability_and_performance/ztp-far-edge/ztp-deploying-far-edge-clusters-at-scale.adoc
```

Comment:
This module does not seem to be included in the assembly reference here. It is included in `scalability_and_performance/ztp-far-edge/ztp-preparing-the-hub-cluster.adoc`. You could update this line to reference that assembly instead.